src/rdb/rdb_raft.c

```diff
@@ -1553,6 +1620,11 @@ rdb_raft_dequeue_event(struct rdb *db, struct rdb_raft_event *event)
 			sizeof(db->d_events[0]) * db->d_nevents);
 }
 
+/* kccain - raft calls:
+ *  raft_is_leader()
+ *  raft_get_current_term()
+ *  raft_become_follower()
+ */
 static void
 rdb_raft_process_event(struct rdb *db, struct rdb_raft_event *event)
 {
```

Comment:
Here it's a bit tricky. We need to make sure we're not holding d_raft_mutex when calling dc_step_up, whose implementation may block and call rdb_tx_begin, etc. (Not applicable to dc_stop, as its implementation is required to _schedule_ but not really execute an rsvc shutdown.)